tests/garage/tf/models/test_categorical_gru_model.py

```diff
@@ -0,0 +1,100 @@
+import pickle
+
+import numpy as np
+import tensorflow as tf
+import tensorflow_probability as tfp
+
+from garage.tf.models import CategoricalGRUModel
+from tests.fixtures import TfGraphTestCase
+
+
+class TestCategoricalGRUModel(TfGraphTestCase):
```

Comment:
this seems significantly less thorough than the other model tests -- am i missing something?

Reply:
same comment for several tests below (no parameterization, one test case, etc.).

i think we should hold these models to the same standard as all the others.

Reply:
You're right, will add more tests.

Reply:
indeed, I realized the reason is that most of the functionality would be exactly the same as GRU, hence only those added on top of GRU (e.g. output a Categorical distribution) are tested.